docker-compose.yml

```diff
@@ -1,6 +1,10 @@
-version: "3.4"
-
+version: '3'
 services:
-  postgresdata:
-    image: ionx/postgres-plv8:12.2
-    command: ["postgres", "-c", "log_statement=all"]
\ No newline at end of file
+  postgresql:
+    image: "ionx/postgres-plv8:12.2"
+    ports:
+     - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_DB: marten_testing
```

Comment:
This was the original `docker-compose.yml` file that was there and works fine, so I reverted my changes.